types/common.tsx

```diff
@@ -1,7 +1,19 @@
 // Type aliases
 export type I18nString = string
 
-// Type interfaces for API gateway result
+// Types for TransLine component
+export interface TransLineProps {
+  i18nKey: I18nString
+  links?: string[] | null
+}
+
+// Types for translation file JSON
+export interface TextOptionalLink {
+  text: string
+  links?: string[]
+}
```

Comment:
This name is pretty terrible: `TextOptionalLink`. Please feel free to suggest alternatives!

Reply:
`LinkableText`?

Reply:
`HereThereBeLinkMaybe`? I don't feel like I have a good one 🤷🏼

Reply:
I'm going to stick with `TextOptionalLink` for this PR. Let's talk synchronously about it if anyone feels strongly about renaming this.